shell/platform/android/test/io/flutter/embedding/engine/FlutterJNITest.java

```diff
@@ -255,4 +256,25 @@ public void setRefreshRateFPS_callsUpdateRefreshRate() {
     // --- Verify Results ---
     verify(flutterJNI, times(1)).updateRefreshRate();
   }
+
+  @Test
+  public void getScaledFontSizeWorks() {
+    final FlutterJNI flutterJNI = new FlutterJNI();
+
+    // Returns -1 below Android U.
+    if (Build.VERSION.SDK_INT < 34) {
```

Comment:
Is this the normal pattern for testing on api versions? I would have expected an annotation declaring what apis this test should be run on. Basically I am surprised that this test can fail based on what api level it is runing against instead of defining what apis can run the test.